docs/user-guide/deploying-application/triggering-ci.md

```diff
@@ -22,13 +22,51 @@ CI builds can be time-consuming for large repositories, especially for enterpris
 * Faster software releases
 * Improved productivity
 
-Get in touch with us if you are looking for a way to improve the efficiency of your software development process
-{% endhint %}
+Get in touch with us if you are looking for a way to improve the efficiency of your software development process.
 
 The **Refresh** icon updates the Git Commits section in the CI Pipeline by fetching the latest commits from the repository. Clicking on the refresh icon ensures that you have the most recent commit available.
 
 The **Ignore Cache** option ignores the previous build cache and creates a fresh build. If selected, will take a longer build time than usual.
 
+---
+
+## Passing Build Parameters [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have [Build & deploy permission](../global-configurations/authorization/user-access.md#role-based-access-levels) or above (along with access to the environment and application) to pass build parameters.
+{% endhint %}
+
+If you wish to pass runtime parameters for build job, you can provide key-value pairs before triggering the build. This will inject those key-value pairs as environment variables in CI runner pods and all its containers.
```

Comment:
Can we try re-writing this in generic terms not how we do it.. Maybe like you can access the passed values with keys from your environment variables...

Reply:
Done